css-layout-classes/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<html>
+<head>
+  <title>CSS Layout</title>
+  <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+
+<div class="row">
+
+  <div class="column left"></div>
+
+  <div class="column middle">
+    <div class="container">
```

Comment:
The `container` class should hold rows and columns, it shouldn't be inside of a `row` or `column` class element.